x-pack/plugins/ml/public/locator/formatters/aiops.ts

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { setStateToKbnUrl } from '@kbn/kibana-utils-plugin/public';
+import { ML_PAGES } from '../../../common/constants/locator';
+import type { ChangePointDetectionUrlState } from '../../../common/types/locator';
+
+/**
+ * Creates URL to the Change Point Deteciton page
```

Comment:
Typo - `Detection`

Reply:
Fixed in https://github.com/elastic/kibana/pull/176694/commits/0543699a73e26183efdd84ee6d13269b34e53a5c